### DISCUSSION TITLE
Completed repository sanitization

### DISCUSSION THREAD

**Discussion by tt-rkim** [tenstorrent/tt-metal#3053] [Contributor announcements]
Hi everyone,

We have completed repository sanitization. All write access for full-time Tenstorrent developers has been restored. Please let us know if you have any issues with access going forward. Furthermore, please ensure that you have deleted all local copies of the repository as they contain the sensitive information that this exercise was supposed to delete.

If you are part of the MCW, Syrmia, Moreh, or other external team, your write access is still revoked. When you have deleted all local copies of the repository, please let us know and we will restore your access.

Thank you for your patience, and goodnight
Raymond

cc: @tenstorrent-metal/developers @tenstorrent-metal/external-developers @tenstorrent-metal/internal-evaluators

**Comment by razorback3** [comment]
Team moreh has completed deleting all local copies.
Please restore the write permission.
Thanks!

**Comment by nemanjagrujic** [comment]
Syrmia has completed deleting all local copies.
Please revoke the write permission.

Regards,
Nemanja.

**Comment by saichandax** [comment]
Team MCW has deleted all local copies of the tt-metal repo.
Kindly revoke the write permissions.
Thank you.

[image: Logo] <https://www.multicorewareinc.com/>
Saichand Akella
Senior Software Engineer
+91 6305188549
Chennai, India.
[image: Facebook icon] <https://www.facebook.com/multicoreware>  [image:
LinkedIn icon] <https://www.linkedin.com/company/multicoreware-inc/>  [image:
Twitter icon] <https://twitter.com/MulticoreWare>  [image: Youtube icon]
<https://www.youtube.com/channel/UCXZ1A1MzS5JwBqwBkNfsBBw>

**Comment by tt-rkim** [comment]
@tenstorrent-metal/syrmia @tenstorrent-metal/mcw @tenstorrent-metal/moreh 
All of your teams have had their write access restored. Please let us know if you see issues still.
